openupgrade_scripts/scripts/account/16.0.1.2/pre-migration.py

```diff
@@ -0,0 +1,563 @@
+from openupgradelib import openupgrade
+
+_xmlids_renames = [
```

Comment:
Hi @ngochung207 Thanks for your work !
I just tested it and found out that this rename is not done (since not called in migrate function below AFAICT).

You may want to add a `openupgrade.rename_xmlids(env.cr, _xmlids_renames)` to enforce it !

Reply:
I know, i have postpone the work from this PR, you may want to try at https://github.com/Viindoo/OpenUpgrade , my work has finished there